spec/lib/pivotal-tracker/account_spec.rb

```diff
@@ -50,7 +50,7 @@
     end
 
     it "can add a new membership" do
-      membership_params = {:email => Faker::Internet.email, :name => Faker::Name.name, :initials => "AW"}
+      membership_params = {:email => 'laney.hoeger@senger.name', :name => 'Larue Mohr', :initials => 'LM'}
       new_membership = (account.memberships.create(membership_params))
```

Comment:
This might be a good place for another factory once we get that branch merged in.